src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,20 @@
+.watch {
+  height: $watch-size;
+  width: $watch-size;
+  border: 5px dotted black;
+  border-radius: 50%;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+
+  &__center {
```

Comment:
this should be a modifier class for `watch__center` something like `watch__center--alignment`